src/pages/overview/pdf-services-api/howtos/combine-pdf.md

```diff
@@ -85,54 +85,89 @@ Please refer the [API usage guide](../api-usage.md) to understand how to use our
 // cd CombinePDF/
 // dotnet run CombinePDF.csproj
```

Comment:
not related to this file, in PDF Services -> -> howtos- > overview -> net-timeout-configuration

please update the latest information / code snippet

Reply:
also remove reference of processing timeout.

Reply:
Fixed